### PR TITLE
[14.0][MIG] stock_inventory_discrepancy

### PR DESCRIPTION
**Pull request by MateuGForgeFlow**
Standard migration to v14.
cc @LoisRForgeFlow @JoanSForgeFlow

### CODE REVIEW

File: stock_inventory_discrepancy/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "summary": "Adds the capability to show the discrepancy of every line in "
     "an inventory and to block the inventory validation when the "
     "discrepancy is over a user defined threshold.",
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.1.0",
```
Comment (rousseldenis) [inline stock_inventory_discrepancy/__manifest__.py]: You can reset to 14.0.1.0.0

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_inventory_discrepancy/__manifest__.py (hunk 1 above)
Comment (MateuGForgeFlow) [reply stock_inventory_discrepancy/__manifest__.py]: Done!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: stock_inventory_discrepancy/views/stock_inventory_view.xml [hunk 2]
```diff
@@ -50,10 +50,10 @@
             </field>
         </field>
     </record>
-    <record id="stock_inventory_line_tree2" model="ir.ui.view">
+    <record id="stock_inventory_line_tree_no_product_create" model="ir.ui.view">
```
Comment (LoisRForgeFlow) [inline stock_inventory_discrepancy/views/stock_inventory_view.xml]: why do you change the ID of the view?

**Review by LoisRForgeFlow — APPROVED** [review]
Ty!

**Pull request merged**